packages/server/src/db/migrations/20230501110838-CreateUserTable.js

```diff
@@ -1,17 +1,41 @@
-const { UserModelDefinition } = require('../../app/user/index.js');
```

Comment:
Why not use the one in the user model?

Reply:
Because the file where it is located now uses import/export, which sequelize doesn't like, because it for some reason doesn't like that..

Reply:
I think I have it working on my branch. Let's do it on my branch

Reply:
ALright, I'll see the changes soon :))